contentcuration/contentcuration/utils/publish.py

```diff
@@ -810,10 +810,12 @@ def mark_all_nodes_as_published(channel):
     logging.info("Marked all nodes as published.")
 
 
-def save_export_database(channel_id):
+def save_export_database(channel_id, version):
     logging.debug("Saving export database")
     current_export_db_location = get_active_content_database()
-    target_export_db_location = os.path.join(settings.DB_ROOT, "{id}.sqlite3".format(id=channel_id))
```

Comment:
Hi @poju3185 - just one thing to flag here, if you read the issue carefully, you will see that it is desirable to continue to export the channel database to the current location as well. The issue is asking to save the database to both the old path and the new path. This means we can keep an archive of the channel databases, but continue to have the 'latest' with the simple name.

Reply:
I've made the adjustments based on the feedback, and now the database is being saved to both the old and new paths. Thanks for pointing this out!

Reply:
Excellent, this looks like the right change now, I'll test it out!